ethereum/sidecar/server.go

```diff
@@ -152,12 +222,46 @@ func RunServer(
 		server.logger.Info("gRPC server routine stopped")
 	}()
 
+	go func() {
+		// TODO: Only a subset of validators should be attesting AssetsUnlocked
+		//       events. Decide whether we should run this goroutine by calling
+		//       MezoBridge.bridgeValidators() with our Ethereum address.
```

Comment:
Could it be as simple as adding a keystore to the base chain, similar to what we did in [keep-core](https://github.com/threshold-network/keep-core/blob/406cb8954005484665422c9693dc780a7ca2d425/pkg/chain/ethereum/ethereum.go#L37)?
The keystore has an `Address` field telling us what is the address of our account, right @lukasz-zimnoch?
We could then call `MezoBridge.bridgeValidators()` with that address to see if we should be attesting.

Reply:
Probably but this is a seprate task: [TET-1383: Ethereum sidecar: Don't run monitoring and attestation loops for non-bridge validators](https://linear.app/thesis-co/issue/TET-1383/ethereum-sidecar-dont-run-monitoring-and-attestation-loops-for-non). Let's not focus on it here.